src/ml_neutronics/testmain.py

```diff
@@ -38,15 +38,15 @@ def normalize_data(y_data, y_val):
 def plot_data( test_case ):
     x_cells = np.arange(1,11,1)
     
-    plt.figure(1)
+    plt.figure(1e+5+test_case)
```

Comment:
Not sure this is the best way to implement differing figures. 
```suggestion
    plt.figure(1e+5+(test_case*2))
```